src/lib/queryFromFilters.js

```diff
@@ -60,6 +60,8 @@ function addConstraint(query, filter) {
       query.greaterThan(filter.get('field'), filter.get('compareTo'));
       break;
     case 'containsString':
+      query.matches(filter.get('field'), filter.get('compareTo'), 'i');
```

Comment:
Its an undocumented "modifiers" parameter to "matches" function of ParseQuery, 'i' means case insensitive: [https://github.com/parse-community/Parse-SDK-JS/blob/master/src/ParseQuery.js#L714](https://github.com/parse-community/Parse-SDK-JS/blob/master/src/ParseQuery.js#L714l)

In retrospective, looking at the ParseQuery code, seems to be possible to pass 'case insensitive' inside the second parameter too.